lib/item_page.dart

```diff
@@ -1,8 +1,94 @@
 import 'package:flutter/material.dart';
+import 'package:lab4/navigation_bar.dart';
+import 'package:lab4/category.dart';
+import 'package:lab4/state.dart';
+import 'package:provider/provider.dart';
 
 class ItemPage extends StatelessWidget {
+  Category? selectedCategory;
+
+  ItemPage({Key? key, this.selectedCategory}) : super(key: key);
+
   @override
   Widget build(BuildContext context) {
-    return Scaffold(body: Text("Item"));
+    return Consumer<CartModel>(
+        builder: (context, cart, child) => Scaffold(
+              extendBodyBehindAppBar: true,
+              appBar: AppBar(
+                elevation: 0,
+                backgroundColor: Colors.transparent,
+                leading: Padding(
+                  padding: const EdgeInsets.all(8),
+                  child: IconButton(
+                    onPressed: () {
+                      Navigator.pop(context);
+                    },
+                    icon: const Icon(Icons.arrow_back_ios_outlined,
+                        color: Colors.black),
+                  ),
+                ),
+              ),
+              body: Center(
+                child: Container(
+                  color: Colors.amber[50],
+                  padding: const EdgeInsets.only(
+                    top: 40,
+                    left: 30,
+                    right: 30,
+                  ),
+                  child: Column(
+                    children: [
+                      const Padding(padding: EdgeInsets.only(top: 100)),
+                      Column(
+                        children: [
+                          Image.asset(selectedCategory!.imgName + '.jpg'),
```

Comment:
it's better to store images with extensions, instead of using concatenation on the UI